lib/colours.js

```diff
@@ -22,7 +26,7 @@ function getColourFromSass (sass, variableName) {
 
 const palette = function () {
   const paletteSass = parseSCSS('_colours-palette.scss')
-  const paletteMap = paletteSass.variables.find(variable => variable.name === '$_govuk-colour-palette-modern').mapValue
```

Comment:
The Sass [variable `$_govuk-colour-palette-modern` was removed recently](https://github.com/alphagov/govuk-frontend/commit/2a0a53908682d2b04f1e3660f63e766f67bd6515)

But we can still use `$govuk-colours`